engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -190,7 +206,13 @@ class WorkflowManagerActor(config: Config)
     val isRestart = id.isDefined
 
     val startMode = if (isRestart) RestartExistingWorkflow else StartNewWorkflow
-    val wfActor = context.actorOf(WorkflowActor.props(workflowId, startMode, source, config), name = s"WorkflowActor-$workflowId")
+    val wfActor = context.actorOf(WorkflowActor.props(workflowId, startMode, source, config, serviceRegistryActor), name = s"WorkflowActor-$workflowId")
+
+    // We have a valid workflowId for the workflow, send it over to the metadata service
+    pushToMetadataService(workflowId) onFailure {
```

Comment:
FWIW Klang suggests failed.foreach: https://github.com/viktorklang/blog/blob/master/Futures-in-Scala-2.12-part-5.md